caikit_nlp/modules/text_generation/peft_config.py

```diff
@@ -56,6 +58,16 @@ class TuningType(str, Enum):
 
 def resolve_base_model(base_model, cls, torch_dtype):
     if isinstance(base_model, str):
+
+        if get_config().base_models_dir:
+
+            error.value_check(
```

Comment:
as written to validate the `base_model` shouldn't this be applicable whether or not there is a `base_models_dir`? or was the check meant to validate the directory itself?

Reply:
actually this case was meant to check for the case where we are considering to look at it from directory perspective, otherwise we leave it to the transformers library. but there is a miss here in terms of implementation. Good catch

Reply:
update makes more sense now! will wait on unit test additions and a format fix